code/style.css

```diff
@@ -5,5 +5,385 @@ We want you focus A LOT on the styling for this project.
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.kitchen a {
+  border: 2px dotted #24292d;
+  border-radius: 16px;
+  width: 100%;
+  text-decoration: none;
+  color: #000;
+  padding: 16px;
+  margin-bottom: 32px;
+}
+
+.kitchen a div {
+  border: 2px  #d8eee6;
+  border-radius: 16px;
+}
+
+.kitchen a img {
+  width: 80%;
+}
+
+
+/*** INTIRIOR CONTAINER ***/
+
+.interior-container {
+  background: #fff;
+  color: #000;
+  text-align: center;
+  align-items: center;
+  padding: 24px;
+}
+
+.interior {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.interior a {
+  width: 100%;
+  height: 100%;
+  text-decoration: none;
+  color: #000;
+  padding: 16px;
+  margin-bottom: 32px;
+}
+
+.interior a img {
+  width: 50%;
+  height:50%;
+}
+
+.interior a div {
+  border: 2px sold #d8eee6;
+  border-radius: 16px;
+}
+
+.interior-square-images {
+  position: relative;
+  width: 200px;
+  height: 200px;
+  height: auto;
+  overflow: hidden;
+  margin-left: auto;
+  margin-right: auto;
+  object-fit: cover;
+  object-position: center;
+  border-radius: 50%;
+}
+
+.interior-square-images img {
+  width: 100%;
+  height: auto;
+  margin-left: -50px;
+}
+
+
+/*** MEDIA QUERIES ***/
+
+@media (min-width: 650px) {
+  header {
+      justify-content: space-between;
+  }
+
+  nav {
+      display: inline-block;
+  }
+
+  .hamburger {
+      display: none;
+  }
+
+  .welcome-container div {
+      text-align: left;
+  }
+}
+
+@media (min-width: 900px) {
+  .welcome-container .container {
+      flex-direction: row;
+      align-items: center;
+  }
+
+  .welcome-container img {
+      width: 50%;
+  }
+
+  .kitchen a {
+      width: 25%;
+      flex-grow: 1;
+      margin: 16px;
+  }
+
+  .interiors a {
+    width: 25%;
+    flex-grow: 1;
+    margin: 16px;
+  }
+}
+
+h4 {
+  text-transform: uppercase;
+  font-size: 30px;
+  margin-top: 0;
+  line-height: 30px;
+  text-align: center;
+  padding-top: 50px;
+  padding-bottom: none;
+}
+
+* {
+  font-family: Helvetica, sans-serif;
+	font-weight: 400;
+}
+
+input[type=text], input[type=email], textarea{ 
+  width: 80%;
+  padding: 12px;
+  border: 1px solid #ccc;
+  border-radius: 4px;
+  box-sizing: border-box;
+  resize: vertical;
+}
+
+label {
+  padding: 12px 12px 12px 0;
+  display: inline-block;
+}
+
+input[type=submit] {
+  background-color: #ccc;
+  color: white;
+  padding: 12px 20px;
+  border: none;
+  border-radius: 4px;
+  cursor: pointer;
+  }
+
+input[type=submit]:hover {
+  background-color: darkgrey;
+}
+
+.your-message-container {
+  border-radius: 5px;
+  background-color: #f2f2f2;
+  padding: 20px;
+  width: 100%;
+  display:block;
+}
+
+.col-25 {
+  float: left;
+  width: 25%;
+  margin-top: 6px;
+}
+
+.col-75 {
+  float: left;
+  width: 75%;
+  margin-top: 6px;
+}
+
+.row:after {
+  content: "";
+  display: table;
+  clear: both;
+}
+
+.checkbox-container {
+  display: block;
+  position: relative;
+  padding-left: 35px;
+  margin-top: 12px;
+  margin-bottom: 12px;
+  cursor: pointer;
+  font-size: 12px;
```

Comment:
12px is kind of hard to read. Maybe 14px or even 16px would be kinder to the eyes :)